scapy/contrib/gtp.py

```diff
@@ -285,6 +291,46 @@ def guess_payload_class(self, payload):
 }
 
 
+class GTPPDUSessionContainer(Packet):
+    name = "GTP PDU Session Container"
+    fields_desc = [ByteField("ExtHdrLen", None),
+                   BitField("type", 0, 4),
+                   BitField("spare1", 0, 4),
+                   BitField("P", 0, 1),
+                   BitField("R", 0, 1),
+                   BitField("QFI", 0, 6),
+                   ConditionalField(XBitField("PPI", 0, 3), lambda pkt: pkt.P == 1),  # noqa: E501
```

Comment:
We do not accept `noqa` for new code.

Reply:
Thank you for your input. I removed noqa and re-run pep08 check. The pep08 check looks fine now after removing noqa.

Reply:
Thanks!